### PR TITLE
Astyle-P

### PR DESCRIPTION
**Pull request by kevingranade**
Just some typical astyling.

### CODE REVIEW

File: src/ballistics.cpp [hunk 1]
```diff
@@ -287,10 +289,14 @@ dealt_projectile_attack projectile_attack( const projectile &proj_arg, const tri
         if( do_animation && !do_draw_line ) {
             // TODO: Make this draw thrown item/launched grenade/arrow
             if( projectile_skip_current_frame >= projectile_skip_calculation ) {
-                g->draw_bullet( tp, (int)i, trajectory, bullet );
+                <<< <<< < HEAD
+                g->draw_bullet( tp, ( int )i, trajectory, bullet );
+                == == == =
+                    g->draw_bullet( g->u, tp, ( int )i, trajectory, stream ? '#' : '*' );
+                >>> >>> > Astyle - P
```
Comment (Coolthulhu) [inline src/ballistics.cpp]: Bad/unfinished merge?

File: src/string_id_null_ids.cpp [hunk 2]
```diff
@@ -4,40 +4,40 @@
...
+MAKE_NULL_ID2( oter_t, "", 0 );
+MAKE_NULL_ID2( oter_type_t, "", 0 );
+MAKE_NULL_ID2( ter_t, "t_null", 0 );
+MAKE_NULL_ID2( trap, "tr_null" );
+MAKE_NULL_ID2( furn_t, "f_null", 0 );
+MAKE_NULL_ID2( MonsterGroup, "GROUP_NULL" );
+MAKE_NULL_ID2( mission_type, "mission_type" );
+MAKE_NULL_ID2( species_type, "spec_null" );
+MAKE_NULL_ID2( mutation_branch );
+MAKE_NULL_ID2( requirement_data, "null" );
```
Comment (Coolthulhu) [inline src/string_id_null_ids.cpp]: No newline at the end?

**Comment by Coolthulhu** [comment]
Also, needs rebase after #21620

**Comment by kevingranade** [comment]
Meh it's easiest to just re-create it entirely, closing.

**Pull request closed without merging**